govuk-notify-spi/src/main/resources/application.conf

```diff
@@ -0,0 +1,8 @@
+notify {
+  apiKeyPath = ${GOVUK_NOTIFY_API_KEY_PATH}
+  templateIdPath = ${GOVUK_NOTIFY_TEMPLATE_ID_PATH}
+}
+
+ssm {
+  endpoint = "https://ssm.eu-west-2.amazonaws.com"
```

Comment:
As per my other comment, you don't need the ssm.endpoint parameter unless you're testing SSM with wiremock.